pdbfixer/pdbfixer.py

```diff
@@ -354,6 +451,50 @@ def _initializeFromPDBx(self, file):
                 for row in modData.getRowList():
                     self.modifiedResidues.append(ModifiedResidue(row[asymIdCol], int(row[resNumCol]), row[resNameCol], row[standardResCol]))
 
+
+    def _downloadCCDDefinition(self, residueName: str, checkCache: bool = True) -> Optional[CCDResidueDefinition]:
+        """
+        Download a residue definition from the Chemical Component Dictionary.
+
+        This method caches results in the ``PDBFixer`` object.
+
+        Parameters
+        ----------
+        residueName : str
+            The name of the residue, as specified in the PDB Chemical Component
+            Dictionary.
+        checkCache : bool
+            If ``False``, attempt to re-download the CCD entry regardless of
+            what is in the cache. Defaults to ``True``.
```

Comment:
What's the reason for including this flag?  You never specify it when calling this method, and it's a private internal method not meant to be called by users, so the value is never False.

Reply:
This is another leftover from an earlier iteration and I've removed it.